### DISCUSSION TITLE
Allow organizers to open submissions with the speaker's link

### DISCUSSION THREAD

**Discussion by magamig** [pretalx/pretalx#1105] [Ideas] [answered]
Try opening a link such as: [https://example.com/conference/me/submissions/XYZXYZ/](https://example.com/conference/me/submissions/XYZXYZ/) without being the speaker - it gives you a 404.

However, this could be improved for the conference organisers by redirecting them to [https://example.com/orga/event/conference/submissions/XYZXYZ/](https://example.com/orga/event/conference/submissions/XYZXYZ/), instead of giving them the error.

**Comment by rixx** [comment, marked as answer]
This was implemented a while ago – just noting this here in case people stumble upon this request: Organisers will be redirected to the organiser area version of the session, just like suggested here.
